DataTransfer/TransferFiles.py

```diff
@@ -0,0 +1,11 @@
+#python 3.6
+from github import Github
```

Comment:
Did you have to install any packages for this? If so add them to the [requirements.txt](https://note.nkmk.me/en/python-pip-install-requirements/).